static/js/com/dropdown/index.js

```diff
@@ -112,4 +110,19 @@ Dropdown.prototype.select = function (index, emit) {
   emit && this._emitter.emit('select', selectedValue);
 };
 
+function renderTemplate(dropdown) {
+  let items = '';
+
+  for (const [key, value] of Object.entries(dropdown.items)) {
+    items += `<div class="dropdown-item js-item" data-value="${key}">${value}</div>`;
```

Comment:
@nikpachoo, I'm not sure how it works in twig, but it looks like unsafe inline attr and html values without escaping. Do you know how we use it? Version selector?

Reply:
Yeah, the version selector uses the dropdown component. The values are [here](https://github.com/JetBrains/kotlin-web-site/blob/cf8b857afb94c15306d75c97f764d77e2bcf538d/static/js/page/api/api.js#L187).

Reply:
@zoobestik, please take a look

Reply:
For me it's ok if it's only in for api version selector. 👍